src/controllers/trip.controller.js

```diff
@@ -23,6 +22,7 @@ class TripController {
       ...req.body, tripId: req.tripId, userId: req.userData.id, tripType: 'one-way', status: 'pending'
     });
     const { updatedAt, createdAt, ...newTrip } = dataValues;
+    await RequestService.createRequest({ requesterId: req.userData.id, tripId: dataValues.tripId, status: 'pending' });
```

Comment:
I'm curious to know why you are making the changes in this line and lines 55 and 95

Reply:
@adesege once a trip is created, the request associated to that trip have to be immediately created in requests table. So, that functionality(create request of the trip in request table) hasn't been implemented by those who worked on both `one-way, return-trip, multi-city` trips. So when we were presenting, @bdushimi told me not to hardcode the requests in request table once i successfully creates a trip in trip table. He told me to add that functionality to those features(one-way, return-trip, multi-city). The reason why you are seeing me changing those features created by others. Than you

Reply:
You should have created a bug fix for that and implement the changes in a different PR. These changes are beyond the scope of this PR.